libdevcore/SHA3.cpp

```diff
@@ -211,11 +211,14 @@ defsha3(512)
 
 }
 
+unsigned g_sha3Counter = 0;
+
 bool sha3(bytesConstRef _input, bytesRef o_output)
 {
 	// FIXME: What with unaligned memory?
 	if (o_output.size() != 32)
 		return false;
+	++g_sha3Counter;
```

Comment:
Counter might skip some calls when this function is used concurrently. I guess this is not too important for benchmarks though